### PR TITLE
chore(surveys): add API docs for the `questions` field

### PR DESCRIPTION
**Pull request by jurajmajerik**
## Changes

Add API docs for the `questions` field on a Survey.

Context: https://posthog.slack.com/archives/C05M3SMFSAG/p1720980094147449

## Testing
I don't think there's a way to preview this. The website compiles the docs from the repo, so hoping this turns out nicely :)

### CODE REVIEW

**Review by Phanatic — APPROVED** [review]
Magnificent! thanks for doing this.

File: posthog/models/feedback/survey.py [hunk 1]
```diff
@@ -56,7 +56,91 @@ class Meta:
...
+        null=True,
+        help_text="""
+        The `array` of questions included in the survey. Each question must conform to one of the defined question types: Basic, Link, Rating, or Multiple Choice.
+
+        Basic (open-ended question)
+        - `type`: `open`
+        - `question`: The text of the question.
+        - `description`: Optional description of the question.
+        - `descriptionContentType`: Content type of the description (`html` or `text`).
+        - `optional`: Whether the question is optional.
```
Comment (dmarticus) [inline posthog/models/feedback/survey.py:70]: does our API documentation tool autogenerate types for these fields?  If not, I think we should specify the types.  It's not as important for fields that are `string` or `number`, especially when it's obviously a string/number field, but for something like this, would should make it clear that it's a boolean.  

Perhaps an example payload for a complex question could help?

**Review by dmarticus — APPROVED** [review]
Looks good, one comment about either (a) specifying the types for any value where it could be ambiguous (b) providing an example payload for what a complex question might look like, or (c) consider doing both?

File: posthog/models/feedback/survey.py (hunk 1 above)
Comment (jurajmajerik) [reply posthog/models/feedback/survey.py:70]: yeah, great point

**Pull request merged**